source/sass/components/nav.scss

```diff
@@ -219,16 +255,11 @@
         a {
```

Comment:
Similar to another comment, can this be `[class^="nav-link-"]` so we're not targeting specific element types with hyper-specificity?

Reply:
This got cleaned up a bit in https://github.com/mozilla/foundation.mozilla.org/pull/2348 so I'll revisit this as followup.